.travis.yml

```diff
@@ -19,13 +19,11 @@ install:
   - rm composer.lock
   - travis_retry composer update --prefer-dist
 
+jobs:
```

Comment:
adding it here is wrong. `include` was a child of `jobs` (sibling of `allow_failures`). Moving the top-level key withut moving its content makes no sense (`include` is not nested inside `script`, in an invalid way)

Reply:
Yes you're right i missed that